src/pages/login/Login.jsx

```diff
@@ -12,14 +11,32 @@ class Login extends React.Component {
 
   render() {
     return (
-      <Card className="loginCard">
-        <Typography m="15px" variant="h3" textAlign="center">Log in</Typography>
-        <Stack m="20px auto" width="350px" height="200px" spacing={2} direction="column">
-          <TextField id="outlined-basic" label="E-mail" variant="outlined" />
-          <TextField id="outlined-password-input" label="Password" type="password" autoComplete="current-password" />
-          <Button variant="contained">Log In</Button>
-        </Stack>
-      </Card>
+      <Box className="login-container">
```

Comment:
Classes names and ides have to be written in camelCase

Reply:
done